Controller/ResettingController.php

```diff
@@ -100,6 +100,22 @@ public function sendEmailAction(Request $request)
             if (null !== $event->getResponse()) {
                 return $event->getResponse();
             }
+        } else {
```

Comment:
Shouldn't this be `elseif(null !== $user)` since the user could still be null here and creating a `GetResponseUserEvent` with a null user should not happen right?
Another `else` could dispatch a `GetResponseNullableUserEvent` if needed.